chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
```

Comment:
Where is the lowercase `throw` mentioned in the chai docs? I can't find it. Did you find this in the source code or something?